.gitignore

```diff
@@ -1,5 +1,8 @@
 .DS_Store
-code
-/packages
-/swift-doc
-/public
```

Comment:
We should keep this as that's where the files are generated (for example when testing it out locally)

Reply:
In fact we don't need to change the gitignore at all do we? This isn't a Swift package